internal/pkg/routes/mfx/mfx.go

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package mfx
+
+import (
+	"fmt"
+
+	mqtt "github.com/eclipse/paho.mqtt.golang"
+	"github.com/gogo/protobuf/proto"
+	"github.com/mainflux/export/internal/pkg/routes"
+	"github.com/mainflux/mainflux"
+	"github.com/mainflux/mainflux/logger"
+	"github.com/mainflux/senml"
+	"github.com/nats-io/nats.go"
+)
+
+const (
+	// JSON represents SenML in JSON format content type.
+	JSON = "application/senml+json"
```

Comment:
Why are these consts capitalized?

Reply:
like here https://github.com/mainflux/senml/blob/master/senml.go#L22 or here
https://github.com/mainflux/mainflux/blob/master/transformers/senml/message.go#L5
and looks more readable to me